src/css-grids/grids.html

```diff
@@ -8,20 +8,32 @@
 </head>
 <body>
   <div class="grid-container">
-    <div class="box top-text">
+    <div class="box a">
+        <img src="fish.png"class="fishimg">
+      </div>
+    <div class="box b">
+
+    </div>
+    <div class="box c">
```

Comment:
the class name c works but it would be good practice to be more specific. like using `box-c` instead